src/main/java/racingcar/RacingCars.java

```diff
@@ -0,0 +1,59 @@
+package racingcar;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingCars {
+
+    private List<RacingCar> racingCarList;
+
+    public RacingCars() {
+        this.racingCarList = new ArrayList<>();
+    }
+
+    protected RacingCars(List<RacingCar> racingCarList) {
+        this.racingCarList = racingCarList;
+    }
+
+    public boolean registRacingCar(RacingCar racingCar) {
```

Comment:
```suggestion
    public boolean registerRacingCar(RacingCar racingCar) {
```
조심스레 제안드려 봅니다
regist는 typo라고 볼 수도 있을 것 같아요